packages/gvim.rb

```diff
@@ -3,81 +3,63 @@
+      system "sed", "-i", "s|^.*#define SYS_GVIMRC_FILE.*$|#define SYS_GVIMRC_FILE \"#{CREW_PREFIX}/etc/gvimrc\"|", "feature.h"
+      system "autoconf"
+    end
+  end
+
   def self.build
-    system "./configure \
-            --prefix=#{CREW_PREFIX} \
-            --disable-nls \
-            --enable-gui \
-            --with-x \
-            --enable-fontset \
-            --enable-multibyte \
-            --with-tlib=ncurses \
-            --enable-perlinterp \
-            --enable-python3interp \
-            --enable-rubyinterp \
-            --with-python3-command=#{CREW_PREFIX}/bin/python3 \
-            --with-ruby-command=#{CREW_PREFIX}/bin/ruby \
-            --with-features=huge \
-            --disable-selinux"
-    system 'make'
+    system "./configure",
+              "--prefix=#{CREW_PREFIX}",
+              "--localstatedir=#{CREW_PREFIX}/var/lib/vim",
+              "--with-features=huge",
+              "--with-compiledby='Chromebrew'",
+              "--with-x=yes",
+              "--enable-gui=gtk3",
+              "--enable-multibyte",
+              "--enable-cscope",
+              "--enable-perlinterp=dynamic",
+              "--enable-pythoninterp=dynamic",
+              "--enable-python3interp=dynamic",
+              "--enable-rubyinterp=dynamic",
+              "--disable-selinux"
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81"
   end
 
   def self.install
-    tmpdir = "#{CREW_PREFIX}/tmp/vim"
-    configdir = "#{CREW_PREFIX}/.config/vim"
-    configdestdir = "#{CREW_DEST_PREFIX}/.config/vim"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    FileUtils.mkdir_p "#{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/evi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/bin/*gvi* #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/evi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/man/man1/*gvi*.1 #{tmpdir}"
-    system "cp #{CREW_DEST_PREFIX}/share/vim/vim81/gvimrc_example.vim #{tmpdir}"
-    system "rm -rf #{CREW_DEST_PREFIX}"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/gvimrc_example.vim #{CREW_DEST_PREFIX}/share/vim/vim81"
-    system "mv #{tmpdir}/*.1 #{CREW_DEST_PREFIX}/share/man/man1"
-    system "mv #{tmpdir}/* #{CREW_DEST_PREFIX}/bin"
-    system "rm -rf #{tmpdir}"
-    system "compressdoc --gzip -9 #{CREW_DEST_PREFIX}/share/man/man1"
-    FileUtils.mkdir_p "#{configdestdir}"
-    FileUtils.mkdir_p "#{CREW_DEST_HOME}"
-    system "touch #{configdir}/gvimrc #{configdestdir}/gvimrc"
-    system "ln -s #{configdir}/gvimrc #{CREW_DEST_HOME}/.gvimrc"
+    system "make", "VIMRCLOC=#{CREW_PREFIX}/etc", "VIMRUNTIMEDIR=#{CREW_PREFIX}/share/vim/vim81", "DESTDIR=#{CREW_DEST_DIR}", "install"
+
+    # these files are provided by the vim package
```

Comment:
Not all of these files are provided by the vim package.

Reply:
They should be, `make install` will install the same runtime files and man pages regardless of the configure switches.

Reply:
Regarding the overlap, I'll double check this, but the `vim` binary is different when building with GUI support. I actually tried to exclude the common binaries in `gvim` and starting it would just fail with "not enabled at compile".

Reply:
See comment below.  Before we do this, we really need to compare both installs before removing entire directories willy-nilly.

Reply:
I did compare the runtime files, I'll check again for the man pages though.